test/mrp/README.md

```diff
@@ -0,0 +1,52 @@
+# Kafka-MMITSS Infrastructure Testing Deployment Note
+
+This page indicate a step-by-step instruction to deploy the testing on MMITSS Kafka Transceiver for MMITSS MRP (Road-side processor) components. 
+
+### Requirements
+* cda-mmitss repository cloned
+* confluent_kafka
+* python 3
+* pytest
+* docker compose file(.ymal). Original template can be found here: https://github.com/usdot-fhwa-stol/carma-streets/blob/develop/docker-compose.yml
+
+
+## Bring up Kafka container
+- open the docker compose file (.yaml), enter the following command:
+    $ docker-compose up kafka
```

Comment:
When trying to show code you can use "```" as a delimiter as shown in this comment
```shell
docker-compose up kafka
docker exec -it kafka bash
```

Reply:
Please see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks for more information on how to include code blocks in mark down